modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -184,7 +183,7 @@
 	fire_sound = 'modular_skyrat/modules/sec_haul/sound/smg_fire.ogg'
 
 /obj/projectile/beam/laser/hardlight_bullet/pcr
-	damage = 27 // Minimum damage increase to at least base /tg/ laser gun damage.
+	damage = 30 // 11/20/21: Increased from 27 to 30 to improve underperforming guns.
```

Comment:
this buffs the PCR  to a time to down of around 2s, making it double a bulldog with buckshots (5s from 2 tiles), making it un-comparably the best current firearm on station.

Reply:
this is the same damage as an energy sword but also you can shoot it through windows and security has it